tile_generator/config.py

```diff
@@ -61,6 +64,12 @@
 # Normalize Jobs - Ensure that job type, template, and properties are set for
 # every job
 
+def _find_git_root():
```

Comment:
Is this intended to be a weak internal use function? 
https://peps.python.org/pep-0008/#descriptive-naming-styles

Reply:
Thank you 👍

Reply:
I was intending to mark it as "internal use"